api/include/mfxstructures.h

```diff
@@ -2333,6 +2352,63 @@ MFX_PACK_END()
 
 #endif
 
+#if (MFX_VERSION >= 1033)
```

Comment:
hm. Why it's under 1.33 right away?

Reply:
IMHO, this is open question. we can use MFX_VERSION_NEXT too. If this, user need additional CMake option to enable AV1d feature. What is your opinion? @dmitryermilov

Reply:
1.33 was already released in 20.2.0: https://github.com/Intel-Media-SDK/MediaSDK/releases/tag/intel-mediasdk-20.2.0. I clearly don't understand how we can append AV1 to something we have already released. Per my understanding this should be >=1.34.

Reply:
Sure. Currently I just uploaded the refinement based on your other comments. If CI build is passed, I will change this to MFX_VERSION_NEXT.

Reply:
Done.

Reply:
I somehow missed this at all. We need to recall what was the original plan. I remember ARB approved AV1 film grain to be in 1.33. Eventually it didn't appear in 1.33. Plus, I can't find any minutes about the rest (major) part of AV1 API. I mean profiles, levels. @stasefremov , do you remember what the plan was ?